cmd/cli/main.go

```diff
@@ -71,6 +77,17 @@ func main() {
 			logger.Errorf("Failed to enable images in all notifications")
 		}
 	}
+	if *fromUser != "" || *toUser != "" {
```

Comment:
Still incorrect expression as it will be true even if only one of options will be specified `formUser` or `toUser`. In our case we need to check that `fromUser` **and** `toUser` are specified

Reply:
@beevee Am I Right in this assumptions?

Reply:
@beevee There are two options, we make a trusted zone and they do what they need or are forbidden to do stupid things (that is, only from a normal user to a normal user)

Reply:
We can transfer to/from anonymous user. Anonymous user is defined by empty string.